tests/console/install_single_package.pm

```diff
@@ -36,6 +35,10 @@ sub run() {
 
     # ensure that package was installed correctly
     assert_script_run("rpm -q $pkgname");
+    assert_script_run("ci-openqa-tests");
```

Comment:
that seems to be applicable only to the HPC tests repo and therefore should not be in here in this way. How about an idea like in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3143 where can supply a specific repository and a specific test script within that?